bitnami/keycloak/templates/configmap-env-vars.yaml

```diff
@@ -32,6 +32,6 @@ data:
   KEYCLOAK_ENABLE_TLS: {{ ternary "true" "false" .Values.auth.tls.enabled | quote }}
   {{- if .Values.auth.tls.enabled }}
   KEYCLOAK_HTTPS_PORT: {{ .Values.containerPorts.https | quote }}
-  KEYCLOAK_TLS_KEYSTORE_FILE: "/opt/bitnami/keycloak/certs/keycloak.keystore.jks"
-  KEYCLOAK_TLS_TRUSTSTORE_FILE: "/opt/bitnami/keycloak/certs/keycloak.truststore.jks"
+  KEYCLOAK_TLS_KEYSTORE_FILE: {{ ternary "/certs/keystore.jks" "/opt/bitnami/keycloak/certs/keycloak.keystore.jks" .Values.auth.tls.sharedJKS | quote }}
```

Comment:
I have my concerns about the paths. It seems to me that, for consistency, the other mount folder should also be `/opt/bitnami/keycloak`. Maybe `/opt/bitnami/keycloak/shared-certs`?